### ISSUE TITLE
Sony WH-1000XM4 - Works, noise with higher switch volume

### ISSUE THREAD

**Issue by pandavova** [plutooo/nx-btred#24]
First of all, thank you for this, this is really cool.
Im on 12.0.1, AMS 0.19.2, MissionControl 0.5.0 alpha2 and nx-bted 0.9.3.

I had problems getting my headphones to show up at first but I just forgot to turn off airplane mode. Maybe a notification that airplane mode is still on would be nice. 

Then I had problems getting sound through my headphones but I also managed to fix that by starting a game (in my case F3H), opening the Homebrew Menu in Applet Mode, connecting my headphones, turning my headphones off and turning them back on. Then they reconnected and I got sound F3H. 
There are few weird things. There is a small delay, but it's iMo acceptable. 
The other weird thing is the volume control. If I put my volume level to the lowest level I get good sound. If make my volume on my switch higher, I get noise. The higher the volume level on the switch, the more noise I get. 
So my temp. solution for that is setting the switch to the lowest volume level and controlling the volume with my headphones.

**Comment by OperationNT414C** [comment]
Yes, I have a similar issue with my Sony WH-1000XM3, same solution.

**Comment by Argyos** [comment]
My XM3 is stuck in the step, GOT SSP when trying to paring them, applet or no apple mode.

EDIT: Now it connect, important to unpair and pair again, meanwhile solution is paring, shutdown and wait to reconect with the app open 

12.0.3, AMS 0.19.4
MissionControl 0.5.0 pre-release 4
nx-bted-0.9.3
